cob_script_server/package.xml

```diff
@@ -13,28 +13,25 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>actionlib_msgs</build_depend>
-  <build_depend>actionlib</build_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>rostest</build_depend>
-  <build_depend>trajectory_msgs</build_depend>
-
-  <run_depend>actionlib_msgs</run_depend>
-  <run_depend>actionlib</run_depend>
-  <run_depend>cob_light</run_depend>
-  <run_depend>cob_mimic</run_depend>
-  <run_depend>cob_sound</run_depend>
-  <run_depend>control_msgs</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>ipython</run_depend>
-  <run_depend>message_runtime</run_depend>
-  <run_depend>move_base_msgs</run_depend>
-  <run_depend>python-pygraphviz</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>rostest</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>std_srvs</run_depend>
-  <run_depend>tf</run_depend>
-  <run_depend>trajectory_msgs</run_depend>
+  <exec_depend>message_runtime</exec_depend>
+
+  <depend>actionlib_msgs</depend>
+  <depend>actionlib</depend>
```

Comment:
Why is there the `depend` on `actionlib`? Because we build actions? The [docs](http://docs.ros.org/jade/api/catkin/html/howto/format2/building_msgs.html) say you only need `actionlib_msgs`. Or did I miss something?